--- phpPgAdmin-4.2/conf/config.inc.php	Sun Apr  6 03:53:28 2008
+++ phpPgAdmin-4.2_pkg/conf/config.inc.php	Mon Apr  7 00:32:18 2008
@@ -15,7 +15,7 @@
 
 	// Hostname or IP address for server.  Use '' for UNIX domain socket.
 	// use 'localhost' for TCP/IP connection on this computer
-	$conf['servers'][0]['host'] = '';
+	$conf['servers'][0]['host'] = 'localhost';
 
 	// Database port on server (5432 is the PostgreSQL default)
 	$conf['servers'][0]['port'] = 5432;
@@ -70,7 +70,7 @@
 	// will be denied. Only set this false once you have read the FAQ and
 	// understand how to change PostgreSQL's pg_hba.conf to enable
 	// passworded local connections.
-	$conf['extra_login_security'] = true;
+	$conf['extra_login_security'] = false;
 
 	// Only show owned databases?
 	// Note: This will simply hide other databases in the list - this does
@@ -118,7 +118,7 @@
 	$conf['show_oids'] = false;
 	
 	// Max rows to show on a page when browsing record sets
-	$conf['max_rows'] = 30;
+	$conf['max_rows'] = 50;
 
 	// Max chars of each field to display by default in browse mode
 	$conf['max_chars'] = 50;
